dashboard/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Allow: /
```

Comment:
I would have expected to disallow crawling since you have to be logged in anyways to see something.

Reply:
Can you propose a suitable `robots.txt` ?

Reply:
```
User-agent: *
Disallow: /
```

Reply:
Just a suggestion. I don't think that it's too important.

Reply:
1. This is generated by and copied verbatim from the Vue / Vite project in `como-ui`
2. I think this is totally irrelevant, since robots.txt [is not looked for in subdirectories](http://www.robotstxt.org/robotstxt.html) ("Where to put it")
3. Here is a [PR to remove it](https://github.com/ClairBerlin/como-ui/pull/12)